package.json

```diff
@@ -37,8 +37,8 @@
     "@react-native-community/async-storage": "1.3.4",
     "@react-native-community/netinfo": "4.1.5",
     "@svgr/webpack": "4.3.2",
-    "@tensorflow-models/body-pix": "^1.0.1",
-    "@tensorflow/tfjs": "^1.1.2",
+    "@tensorflow-models/body-pix": "^1.1.2",
+    "@tensorflow/tfjs": "^1.2.9",
```

Comment:
While you're at it could get rid of the caret? Not that it really matters now but for the sake of consistency with the rest of package.json.